develop/toolkit/api-libraries/polkadart.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Polkadart
+description: Polkadart is a type-safe, native Dart, SDK for Polkadot and any compatible Polkadot-SDK blockchain network.
+categories: Tooling, Dapps
+---
+
+# Polkadart
+
+Polkadart is the most comprehensive Dart/Flutter SDK for interacting with Polkadot, Substrate, and other compatible blockchain networks. Designed with a Dart-first approach and type-safe APIs, it provides everything developers need to build powerful decentralized applications.
+
+## Installation
+
+Add Polkadart to your `pubspec.yaml`:
+
+=== "All packages"
+
+    ```bash
+    dart pub add polkadart polkadart_cli polkadart_keyring polkadart_scale_codec secp256k1_ecdsa sr25519 ss58 substrate_bip39 substrate_metadata
+    ```
+
+=== "Core only"
+
+    ```bash
+    dart pub add polkadart polkadart_cli polkadart_keyring
+    ```
+
+For type-safe API generation, add the following to your `pubspec.yaml`:
+
+```yaml
+polkadart:
+  output_dir: lib/generated
+  chains:
+    polkadot: wss://rpc.polkadot.io
+    kusama: wss://kusama-rpc.polkadot.io
+    custom: wss://your-node.example.com
+```
+
+## Quick Start
+
+### 1. Creating an API Instance
```

Comment:
Please do not call this section "Quick Start" and do not use step with numbers as this section cannot be followed step-by-step. There are no instructions on where to copy and paste the code, how to run the code, and the different code snippets are for different use cases. Simply remove the number steps and consider adding a sentence highlighting what each code snippet does.

You can call this section "Get Started" similar to how it is done here:
- https://docs.polkadot.com/develop/toolkit/api-libraries/papi/